### PR TITLE
added questions to general and wallet-bridge FAQs

### PR DESCRIPTION
**Pull request by cerberushades**

### CODE REVIEW

File: docs/faq/wallet-bridge-faq.md [hunk 1]
```diff
@@ -318,4 +318,42 @@ For speeding up transactions via Metamask, please go to this [link](https://meta
...
+If the transaction is still at the **Approve** step, it has not been completed yet. To fulfill it, you need to pay the gas fee and then it should go through.
+
+## The gas fee for my withdrawal transaction is too high.
+
+Gas fees on the Polygon network are always very low, but we can’t say the same for the Ethereum mainnet fees. 
+If you have started the transaction, it is important not to cancel it. Instead, you can wait for it to lower or add more Ether to your account. 
+
+## There are some unauthorized transactions in my wallet. Is my wallet hacked?
+
+As we never ask for your private keys or use them in any way, we can’t revert unwanted transactions.
```
Comment (DannyS03) [inline docs/faq/wallet-bridge-faq.md:332]: Would benefit from rewording as it seems we are claiming that if we did have access to your keys we could revert the transaction

**Review by DannyS03 — CHANGES_REQUESTED** [review]
Thanks @cerberushades, left some comments

File: docs/faq/wallet-bridge-faq.md [hunk 2]
```diff
@@ -318,4 +318,44 @@ For speeding up transactions via Metamask, please go to this [link](https://meta
...
+If the transaction is still at the **Approve** step, it is not yet complete. To fulfill it, you need to pay the gas fee and then it should go through.
+
+## The gas fee for my withdrawal transaction is too high.
+
+Gas fees on the Polygon network are always very low, but we can’t say the same for the Ethereum mainnet fees. 
+If you have started the transaction, it is important not to cancel it. Instead, you can wait for it to lower or add more Ether to your account. 
+
+## There are some unauthorized transactions in my wallet. Is my wallet hacked?
+
+Unfortunately, we can’t revert unwanted transactions.
```
Comment (DannyS03) [inline docs/faq/wallet-bridge-faq.md:332]: would be even more clear here and state "the network cannot revert unwanted transactions"

File: docs/faq/wallet-bridge-faq.md [hunk 3]
```diff
@@ -318,4 +318,44 @@ For speeding up transactions via Metamask, please go to this [link](https://meta
...
 - Screenshots or screen recordings of the action
 
+## I was trying to make a deposit but the transaction stopped at the Approve step.
+
+If the transaction is still at the **Approve** step, it is not yet complete. To fulfill it, you need to pay the gas fee and then it should go through.
+
+## The gas fee for my withdrawal transaction is too high.
+
+Gas fees on the Polygon network are always very low, but we can’t say the same for the Ethereum mainnet fees. 
+If you have started the transaction, it is important not to cancel it. Instead, you can wait for it to lower or add more Ether to your account.
```
Comment (DannyS03) [inline docs/faq/wallet-bridge-faq.md:328]: - unclear about what is waiting to lower (gas fees)
- a user may ask if they need to add more ETH or MATIC, do both work?
- the case for when a user is stuck -> we should clearly state if this is the only option

**Review by DannyS03 — CHANGES_REQUESTED** [review]
few more comments
- should be consistent if bolding the options that appear when transacting, such as Approve, Cancel, etc.
- doc would benefit from formatting (as in, prettify)

**Review by Nazeim — APPROVED** [review]

File: docs/faq/wallet-bridge-faq.md [hunk 4]
```diff
@@ -321,3 +321,43 @@ For speeding up transactions via Metamask, please go to this [link](https://meta
...
+If the transaction is still at the **Approve** step, it is not yet complete. To fulfill it, you need to pay the gas fee and then it should go through.
+
+## The gas fee for my withdrawal transaction is too high.
+
+Gas fees on the Polygon network are always very low, but we can’t say the same for the Ethereum mainnet fees. 
+If you have started the transaction, it is important not to cancel it. Instead, you can wait for it to lower or add more Ether to your account. 
+
+## There are some unauthorized transactions in my wallet. Is my wallet hacked?
+
+Unfortunately, we can’t revert unwanted transactions.
```
Comment (DannyS03) [inline docs/faq/wallet-bridge-faq.md:336]:
```suggestion
Unfortunately, the network cannot revert unwanted transactions. 
```

**Pull request merged**